### PR TITLE
testing/toilet: new aport

### PR DESCRIPTION
**Pull request by Shizmob**
http://caca.zoy.org/wiki/toilet
A free replacement for the FIGlet utility

### CODE REVIEW

File: testing/toilet/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+pkgname=toilet
+pkgver=0.3
+pkgrel=0
+pkgdesc="A free replacement for the FIGlet utility"
+url="http://caca.zoy.org/wiki/toilet"
+arch="all"
+license="WTFPL"
+makedepends="libcaca-dev"
+subpackages="$pkgname-doc"
+source="https://distfiles.alpinelinux.org/distfiles/$pkgname-$pkgver.tar.gz"
```
Comment (jirutka) [inline testing/toilet/APKBUILD]: What, this is not the upstream source…

Comment (Shizmob) [inline testing/toilet/APKBUILD]: The upstream server is extremely flaky, apparently to the extent it already existed on distfiles without me having to request it.

Comment (jirutka) [inline testing/toilet/APKBUILD]: Then use abuild snapshot.

Comment (Shizmob) [inline testing/toilet/APKBUILD]: This is not a git or svn repo, and it won't guarantee a build working if the server is down while a snapshot is being made (initially, for instance).

Comment (Shizmob) [inline testing/toilet/APKBUILD]: The issue is not a changing tarball, but the upstream server being offline/down a lot.

**Comment by clandmeter** [comment]
Closed due to inactivity.

**Pull request closed without merging**